serving/docker/scripts/install_trt_llm.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
```

Comment:
is the plan to use this script in the dockerfile? 
I would suggest we do similar to FT container, where we pull the binaries directly in the dockerfile instead of building each time since it only changes per release and we don't need nightly builds.

Reply:
good points, let's move this to a different place